src/build/simp.spec

```diff
@@ -111,18 +111,19 @@ Prefix: %{_sysconfdir}/puppet
 %package extras
 Summary: Extra Packages for SIMP
 License: Apache-2.0
-Requires: pupmod-puppet-grafana >= 3.0.0-2016
+Requires: pupmod-cristifalcas-journald >= 0.5.0-2016
```

Comment:
Is this correct?

Reply:
@jeefberkey Yeah, it is. It turns out that we never used our fork and RHEL 7 automatically forwards from journald to syslog by default so no harm, no foul.